db/migrate/20180611171416_create_real_sitemaps.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRealSitemaps < ActiveRecord::Migration
+  def change
+    create_table :real_sitemaps do |t|
```

Comment:
Did we intend to change the name of the table to `real_sitemaps`? That seems like a weird choice. It's in the database, so of course it's real...

Reply:
Nope. I commented on this above.

@ethanewing , another good rule of thumb is to consider how readable & understandable your code will be to future developers. If you came across a class named `RealFoo`, you would probable wonder, 'what makes this different from `Foo`? Is there a `FakeFoo` class?`. Confusion would ensue.